packages/tables/backend/sirius-components-tables/src/main/java/org/eclipse/sirius/components/tables/components/LineComponent.java

```diff
@@ -73,30 +74,46 @@ public Element render() {
     }
 
     private Element doRender(VariableManager lineVariableManager, String targetObjectId, Optional<Line> optionalPreviousLine) {
-        LineDescription lineDescription = this.props.getLineDescription();
-        UUID lineId = optionalPreviousLine.map(Line::getId).orElseGet(() -> this.computeLineId(targetObjectId));
+        LineDescription lineDescription = this.props.lineDescription();
+        UUID rowId = optionalPreviousLine.map(Line::getId).orElseGet(() -> this.computeLineId(targetObjectId));
 
         String targetObjectKind = lineDescription.getTargetObjectKindProvider().apply(lineVariableManager);
 
-        var cells = this.getCells(lineVariableManager, lineId);
+        var cells = this.getCells(lineVariableManager, rowId);
+        boolean resizable = lineDescription.getIsResizablePredicate().test(lineVariableManager);
+        Integer initialHeight = lineDescription.getInitialHeightProvider().apply(lineVariableManager);
 
         List<Element> children = new ArrayList<>();
         children.addAll(cells);
 
-        LineElementProps lineElementProps = LineElementProps.newLineElementProps(lineId)
+        LineElementProps.Builder rowElementProps = LineElementProps.newLineElementProps(rowId)
                 .descriptionId(lineDescription.getId())
                 .targetObjectId(targetObjectId)
                 .targetObjectKind(targetObjectKind)
                 .children(children)
-                .build();
-
-        return new Element(LineElementProps.TYPE, lineElementProps);
+                .resizable(resizable)
+                .initialHeight(initialHeight);
+
+        this.props.tableEvents().stream()
```

Comment:
Compute this before hand to keep the builder as is and not allow the introduction of nullable values in the builder

Reply:
Done

Reply:
No it's not